sources/package-lock.json

```diff
@@ -1,6 +0,0 @@
-{
-  "name": "sources",
-  "lockfileVersion": 2,
```

Comment:
It says that this file is deleted.

Reply:
https://github.com/bcgov/SIMS/pull/1750/files#r1115186071

Reply:
Thanks @ann-aot 😉

Reply:
I read the comment earlier but now I got it.